### PR TITLE
Route styling

### PR DESCRIPTION
**Pull request by MrTheGood**

### CODE REVIEW

File: src/components/Multi/Route.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ function Geometry() {
                 _coordinates = coordinates
             }
             else {
-                throw new Error('not a array nigger')
+                throw new Error('Not an array, noob!')
```
Comment (CJVisser) [inline src/components/Multi/Route.js]: Please make error Not an array, contact administrator.

File: src/components/Single/Home/Home.vue [hunk 2]
```diff
@@ -32,7 +32,7 @@
                     </div>
                     <div class="content">
                         We hebben een geweldige trip gehad naar het buitenland. <br>
-                        Zeker een aanrader.
+                        Zeker geen aanrander.
```
Comment (CJVisser) [inline src/components/Single/Home/Home.vue:35]: please change back and if time to english

File: src/components/Multi/Route.js (hunk 1 above)
Comment (CJVisser) [inline src/components/Multi/Route.js]: Please make error Not an array, contact administrator.

File: src/components/Single/Home/Home.vue (hunk 2 above)
Comment (MrTheGood) [reply src/components/Single/Home/Home.vue:35]: This is just some demo content that will be removed as soon as it gets loaded from the database

File: src/components/Multi/Route.js (hunk 1 above)
Comment (MrTheGood) [reply src/components/Multi/Route.js]: done

**Pull request merged**